### ISSUE TITLE
Hard to follow this instructions, it seems that there is some missing steps?

### ISSUE THREAD

**Issue by timothylampung** [rachel-fenichel/blockly-samples#928]
https://blocklycodelabs.dev/codelabs/custom-generator/index.html?index=..%2F..index#2

"Generate code
Next, register your new generator with the playground. Find the configurePlayground function in advanced_playground.html and add this line:"

**Comment by alschmiedt** [comment]
Hello!

Sorry for the delay in getting back to you. Can you provide a bit more information on what was confusing? Or what part of this was difficult for you to finish?

**Comment by maribethb** [comment]
There is a new version of the generator codelab that doesn't rely on patching the playground code in core. Closing this as obsolete. Thanks!
